### PR TITLE
Gravity Wiz // Enable Decimal Values for Quantity Fields

### PR DESCRIPTION
**Pull request by sbassah**
Original Gist: https://gist.github.com/spivurno/3710739

### CODE REVIEW

File: gravity-forms/gw-quantity-as-decimal.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Accept Decimal Values for Quantity Fields
+ *
+ * Allows you to accept decimal values in Quantity fields, converting any Quantity field into a weight field.
+ *
+ * @version   1.3
+ * @author    David Smith <david@gravitywiz.com>
+ * @license   GPL-2.0+
+ * @link      http://gravitywiz.com/enable-decimal-values-in-quantity-fields/
```
Comment (spivurno) [inline gravity-forms/gw-quantity-as-decimal.php:10]: Can you replace this with a plugin header?

**Review by spivurno — CHANGES_REQUESTED** [review]

File: gravity-forms/gw-quantity-as-decimal.php (hunk 1 above)
Comment (sbassah) [reply gravity-forms/gw-quantity-as-decimal.php]: @spivurno  I have updated the snippet with a plugin header.

Comment (spivurno) [reply gravity-forms/gw-quantity-as-decimal.php]: Ah, my intent was only to replace the lines that started with `@` that have been highlighted above. Could you put the other stuff back?

**Review by spivurno — APPROVED** [review]
Good work, my friend!

**Pull request merged**